docs/api/app.md

```diff
@@ -396,6 +396,21 @@ You can request the following paths by the name:
 * `videos` Directory for a user's videos.
 * `pepperFlashSystemPlugin`  Full path to the system version of the Pepper Flash plugin.
 
+### `app.getFileIcon(path, [options], callback)`
+
+* `path` String
+* `options` Object (optional)
+  * `size` String - Can be `small`, `normal`, `large`. The `large` size is not supported on _macOS_
```

Comment:
I have a no idea tbh, and no Apple device to test it.
> what could be the best behaviour?

I think the best would be to fetch normal one. This is what happens on windows too, where normal and large icons are equal.

I'll see what I can do to test it